lib/node_modules/@stdlib/math/base/special/sincospi/test/test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var sincospi = require( './../lib/' );
+
+
+// FIXTURES //
+
+var integers = require( './fixtures/julia/integers.json' );
+var decimals = require( './fixtures/julia/decimals.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof sincospi, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'if provided negative infinity, the function returns `[NaN,NaN]`', function test( t ) {
+	var y = sincospi( NINF );
+	t.equal( isnan( y[0] ), true, 'returns NaN when provided negative infinity' );
+	t.equal( isnan( y[1] ), true, 'returns NaN when provided negative infinity' );
+	t.end();
+});
+
+tape( 'if provided positive infinity, the function returns `[NaN,NaN]`', function test( t ) {
+	var y = sincospi( PINF );
+	t.equal( isnan( y[0] ), true, 'returns NaN when provided positive infinity' );
+	t.equal( isnan( y[1] ), true, 'returns NaN when provided positive infinity' );
+	t.end();
+});
+
+tape( 'if provided `NaN`, the function returns `[NaN,NaN]`', function test( t ) {
+	var y = sincospi( NaN );
+	t.equal( isnan( y[0] ), true, 'returns NaN when provided NaN' );
+	t.equal( isnan( y[1] ), true, 'returns NaN when provided NaN' );
+	t.end();
+});
+
+tape( 'the function computes results in-place and returns a reference to the input array', function test( t ) {
+	var out = [ 0.0, 0.0 ];
+	var y = sincospi( out, 0.0 );
+	t.deepEqual( y, [ 0.0, 1.0 ], 'computes the correct result' );
+	t.equal( y, out, 'returns a reference to the input array' );
+	t.end();
+});
+
+tape( 'the function computes the sine and cosine for integer input', function test( t ) {
```

Comment:
Test description needs updating.

Reply:
`computes sin(πx) and cos(πx) for integer input`?

Reply:
Looks good to me, thanks!